src/problems/ql_logistic_centered_exp.jl

```diff
@@ -158,14 +158,15 @@ function QLLogisticCenteredExp(
     design = hcat(ones(T, nobs), randn(T, nobs, nvar-1) ./ T(sqrt(nvar - 1)))
 
     # get reponses
-    β_true = randn(T, nvar)
+    β_true_mean = randn(T, nvar)
```

Comment:
Are you using `β_true_mean` in another part of the code? Otherwise, `β_true` can just be normal random variable with mean zero and variance 2, no?